lib/line_item.go

```diff
@@ -176,7 +182,8 @@ type CreateLineItemCriteria struct {
 	SurveyURL           *string           `json:"surveyURL,omitempty" valid:"optional,surveyURL"`
 	SurveyTestURL       *string           `json:"surveyTestURL,omitempty" valid:"optional"`
 	IndicativeIncidence float64           `json:"indicativeIncidence" valid:"required"`
-	DaysInField         int64             `json:"daysInField" valid:"required"`
+	DaysInField         int64             `json:"daysInField" valid:"optional"`
```

Comment:
Can we add basic validations like one of these must be provided. We can avoid service calls for the basic things which we know will give out an error.

Reply:
Done.